core/octobot.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, config, ignore_config=False, reset_trading_history=False):
             BOT_TOOLS_RECORDER: None,
         }
 
+        # metrics if enabled
+        self.metrics_handler = None
```

Comment:
Why not in Initializer ?

Reply:
To me it seems more legit to store it in octobot

Reply:
Ok :+1: